ext/node/polyfills/internal/crypto/diffiehellman.ts

```diff
@@ -106,41 +123,72 @@ export class DiffieHellman {
     outputEncoding: BinaryToTextEncoding,
   ): string;
   computeSecret(
-    _otherPublicKey: ArrayBufferView | string,
+    otherPublicKey: ArrayBufferView | string,
     _inputEncoding?: BinaryToTextEncoding,
     _outputEncoding?: BinaryToTextEncoding,
   ): Buffer | string {
-    notImplemented("crypto.DiffieHellman.prototype.computeSecret");
+    const sharedSecret = ops.op_node_dh_compute_secret(
+      this.#prime,
+      this.#privateKey,
+      Buffer.from(otherPublicKey.buffer),
+    );
+    return Buffer.from(sharedSecret.buffer);
   }
 
   generateKeys(): Buffer;
   generateKeys(encoding: BinaryToTextEncoding): string;
   generateKeys(_encoding?: BinaryToTextEncoding): Buffer | string {
-    notImplemented("crypto.DiffieHellman.prototype.generateKeys");
+    const generator = this.#generator.readUint32BE();
```

Comment:
Assuming that generators are <= 32 bits, we should probably add a warning somewhere.

I have no knowledge of anyone using DH with a generator of anything other than 2 or 5.

Reply:
I think we should hard error for anything other than 2 or 5.